zaproxy/zaproxy-website
# FAQ updates

Update FAQ content in:
 - `Can ZAP be used to test my favorite vulnerable app?`
   - Tweak text, `repo` → `site`.
 - `How can I prevent ZAP from sending me 1000s of emails via a 'Contact
Us' form?`
   - Link to help pages.
 - `How can I run ZAP with a high DPI display?`
   - Use image from the site.
 - `How can I use the ZAP API in my own regression tests?`
   - Replace API endpoint explanation with a link to API docs.
 - `How can you start ZAP?`
   - Use h3 for the OSes, h2 is used for the FAQ title;
   - Correct indentation of Windows start menu.
 - `How can you use ZAP to scan APIs?`
   - Link to the help of the add-ons;
   - Update OpenAPI entry and move to top, the OpenAPI add-on is already
available in the marketplace;
   - Add a note that they are available in the marketplace after
enumerating the add-ons instead of adding in each one.
 - `How can ZAP automatically authenticate via forms?`
   - Link to API docs instead of duplicating in the FAQ.
 - `How can ZAP test sites that use certificate pinning?`
   - Remove misplaced parentheses;
   - Link just once to the wiki page.
 - `How do I handle a False Positive?`
   - Remove reference to "Context" when mentioning alert filters, it now
supports global alert filters.
 - `How do I see what version of an add-on/extension I have installed?`
   - Update text and image (thanks @kingthorin :) to match the new
layout in Manage Add-ons dialogue;
   - Use h3 for the ways, h2 is used for the FAQ title.
 - `How do you add a script to ZAP from the command line?`
   - Link to command line help page.
   - Remove intro about the keys, link to the FAQ that explains how to
know the keys.
 - `How do you configure ZAP logging?`
   - Update link to source code;
   - Link to help of the add-on when mentioning the script type.
 - `How do you find out what key to use to set a config value on the
command line?`
   - Link to command line help page.
 - `Setting up ZAP to Test Vaadin Apps`
   - Remove sub-title, redundant with main title.
 - `Somethings not working. What should I do?`
   - Tweak text that links to online documentation, it's not just the
main help it also has the help of the add-ons;
   - Link to JVM option help page, which explain how to increase/change
the JVM memory from within ZAP.
 - `What 'calls home' does ZAP make?`
   - Mention new 2.9.0 CFU change;
   - Update tense in some entries, it's already happening or the command
line option is already available.
   - Link to marketplace page.
 - `What does ZAP test for?`
   - Mention WebSockets passive scanning;
   - Link to more add-ons that provide scan rules.
 - `What is the default directory that ZAP uses?`
   - Mention that dev builds use the same dir as the weekly.
 - `What versions of Java are supported?`
   - Mention that ZAP is able to run with all/newer Java versions.
 - `Why has the Quick Scan Attack reported an invalid URL?`
   - Use h3 for HTTPS, h2 is used for the FAQ title.
 - `Why is an API key required by default?`
   - Lower case the version word;
   - Reword a phrase to make clear that's the following option that's
not recommend.
